### PR TITLE
[API Deprecation]Change DGLHeteroGraph to DGLGraph in DOC

### PR DESCRIPTION
**Pull request by peizhou001**
## Checklist
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [ ] I've leverage the [tools](https://docs.google.com/document/d/1iHyj7zlmygKSk5gBPsqIqL5ASPzJSPREaNT_QdsiYA4/edit) to beautify the python and c++ code.
- [ ] The PR is complete and small, read the [Google eng practice (CL equals to PR)](https://google.github.io/eng-practices/review/developer/small-cls.html) to understand more about small PR. In DGL, we consider PRs with less than 200 lines of core code change are small (example, test and documentation could be exempted).
- [ ] All changes have test coverage
- [ ] Code is well-documented
- [ ] To the best of my knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change
- [ ] Related issue is referred in this PR
- [ ] If the PR is for a new model/paper, I've updated the example index [here](../examples/README.md).

### CODE REVIEW

File: src/graph/serialize/heterograph_serialize.cc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /**
  *  Copyright (c) 2019 by Contributors
  * @file graph/serialize/heterograph_serialize.cc
- * @brief DGLHeteroGraph serialization implementation
```
Comment (mufeili) [inline src/graph/serialize/heterograph_serialize.cc:4]: Let's not modify C/C++ files for the time being.

**Review by mufeili — CHANGES_REQUESTED** [review]
One minor comment

File: src/graph/serialize/heterograph_serialize.cc [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+ * @brief DGLGraph serialization implementation
  *
  * The storage structure is
  * {
```
Comment (frozenbugs) [inline src/graph/serialize/heterograph_serialize.cc:7]: In Line 11, there is kDGLHeteroGraph, should that be replaced or already outdated?

Comment (mufeili) [reply src/graph/serialize/heterograph_serialize.cc:7]: I suggest not touching `.cc` files for the time being.

File: src/graph/serialize/heterograph_serialize.cc (hunk 1 above)
Comment (peizhou001) [reply src/graph/serialize/heterograph_serialize.cc:4]: Make sense. Let me revert it.

File: src/graph/serialize/heterograph_serialize.cc (hunk 2 above)
Comment (peizhou001) [reply src/graph/serialize/heterograph_serialize.cc:7]: Revert C code.

**Review by frozenbugs — APPROVED** [review]

**Pull request merged**